web-components/src/components/tabs/Tabs.ts

```diff
@@ -58,6 +58,7 @@ export namespace Tabs {
     // tabsId and persistSelection attributes are used to persist the selection of tab on remount of md-tabs component
     @property({ type: String, attribute: "tabs-id" }) tabsId = "";
     @property({ type: Boolean, attribute: "persist-selection" }) persistSelection = false;
+    @property({ type: String, attribute: "comp-unique-id" }) compUniqueId = "";
```

Comment:
Is this very different from **tabsId** or is it possible to use that? If not can we rename this?

Reply:
tabsId will be updated with taskId. compUniqueId will be static and required in the clear event as well.

Reply:
What is **taskId**?

Reply:
taskid is passed by desktop, this can change for each contact offer. hence we needed a unique+static id